example/example.rs

```diff
@@ -0,0 +1,15 @@
+#![feature(no_core)]
```

Comment:
Maybe name this file `basic_math.rs` or something, rather than just `example` since it's already in that directory.

Also `example` -> `examples` for the directory name, since there will be more than one :)